gweatherrouting/main.py

```diff
@@ -33,11 +33,11 @@ def startUIGtk():
     gi.require_version("Gtk", "3.0")
     from gi.repository import Gtk
 
-    from .gtk.mainwindow import MainWindow
+    from gweatherrouting.gtk.mainwindow import MainWindow
 
     MainWindow(Core())
     Gtk.main()
 
 
-# if __name__ == "__main__":
-#     startUIKivy()
+if __name__ == "__main__":
+    startUIGtk()
```

Comment:
this is not necessery, since the "gweatherrouting" entry points already point to startUIGtk in `setup.py` (https://github.com/dakk/gweatherrouting/blob/master/setup.py#L67C13-L67C63)

Reply:
Right indeed. It's just form your mention that the development should be focussed on the gtk, I thought it would be best to be uncommented by default. But perhaps not!

Reply:
You are right, but we will move away the kivy part entirely in a separate pull request 👍